src/components/StudyPane/InfoPane/Motif/index.tsx

```diff
@@ -16,6 +16,22 @@ const Motif = ({
 
   const activeClasses = "text-primary border-primary";
   const inactiveClasses = "border-transparent";
+  let rootWordsMap = new Map<number, HebWord[]>();
```

Comment:
Can you move this block back into Root component? Just pass content into Root like before